spec/acceptance/scl_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper_acceptance'
```

Comment:
My biggest worry about this test is that it's silently running with the non-SCL version. There is no cleanup before or after. Perhaps it needs a `before(:context)` and `after(:context)` where it removes both the SCL and non-SCL versions.

Reply:
good point - I've added some code for this but I'm very unvertain if it is correct. Could you check please?